src/CatalogDetail/CatalogDetail.js

```diff
@@ -5,39 +5,25 @@ import CatalogSection from '../Section/CatalogSection/CatalogSection'
 import StatisticsSection from '../Section/StatisticsSection/StatisticsSection'
 import OrganizationsSection from '../Section/OrganizationsSection/OrganizationsSection'
 import HarvestsSection from '../Section/HarvestsSection/HarvestsSection'
+import fetchCatalog from '../fetch/fetchCatalog'
+import fetchMetrics from '../fetch/fetchMetrics'
 
 class CatalogDetail extends Component {
   constructor(props) {
     super(props)
     this.state = {catalog: undefined, metrics: undefined}
```

Comment:
Bad practice.
Consider using `getInitialState()`.
Moreover setting values to `undefined` can be easily work-arounded.